client/src/schema/index.test.ts

```diff
@@ -243,7 +246,7 @@ describe('request', () => {
       };
     });
 
-    await expect(users.getOne()).rejects.toThrowErrorMatchingInlineSnapshot(`"Not Found"`);
+    await expect(users.getOne()).rejects.toThrowErrorMatchingInlineSnapshot('"expected is not a function"');
```

Comment:
My bad. Not for self to review this

Reply:
Happens don't worry, this is why we have review 😉